src/main/java/org/candlepin/subscriptions/ApplicationProperties.java

```diff
@@ -152,6 +152,6 @@
   /** Additional properties related to the Subscription Service */
   private SubscriptionServiceProperties subscription = new SubscriptionServiceProperties();
 
-  /** If enabled, will sync Offerings with the upstream product service. */
-  private boolean offeringSyncEnabled = false;
+  /** If enabled, will sync Subscriptions with the upstream subscription service. */
+  private boolean subscriptionSyncEnabled = false;
```

Comment:
Both `offeringSyncEnabled` and `subscriptionSyncEnabled` should exist, right? They're two independent (though related) things I think.

Reply:
They are, but during our chat yesterday, Kevin said I could rename the `offeringSyncEnabled` as it's currently not being used afaik. It can probably be added back in when used.

Reply:
We need only a single var. We either want to sync subscription/offering information from RHIT services, or we want to use candlepin pools.

Reply:
@kahowell In that case, can I leave this variable as is and I will add a story to control the offering sync via this config?

Reply:
:+1: